src/components/refactor/dialogs/load-creation/load-creation-dialog.js

```diff
@@ -234,7 +236,8 @@ const LoadCreationDialog = ({
 
 LoadCreationDialog.propTypes = {
     editData: PropTypes.object,
-    currentNodeUuid: PropTypes.string,
+    studyUuid: PropTypes.string,
+    currentNode: PropTypes.string,
```

Comment:
change type to object instead of string
```suggestion
    currentNode: PropTypes.object,
```